bar.cpp

```diff
@@ -0,0 +1,18 @@
+#include "bar.hpp"
+
```

Comment:
Remove empty line

Reply:
It is incorrect according to [code style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)